integration/test_utils.go

```diff
@@ -48,28 +51,61 @@ var (
 )
 
 func init() {
+	if err := ConnectDaemons(); err != nil {
+		glog.Exitf("Failed to connect daemons: %v", err)
```

Comment:
Failed to connect with containerd..

Reply:
This tries to connect cri-containerd and containerd, not only containerd.

Reply:
"Failed to connect cri-containerd and containerd"?